### PR TITLE
Feature/us5834 updateaccountingviews

### PR DESCRIPTION
**Pull request by lknair2015**

### CODE REVIEW

File: CI/SQL/20161201_150512_US5834_createcrdsdepositexportpreviewreportinmp.sql [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+				[On_Reports_Tab]
+			   )
+		 VALUES
+			   (305
+			   ,N'Selected Deposit - Export Preview'
+			   ,N'Selected Deposit - Export Preview (Donations and Payments)'
+			   ,N'/MPReports/Crossroads/CRDS Selected Deposit Export Preview'
+			   ,1
+			   ,1          
+			   ,0)
```
Comment (silbermm) [inline CI/SQL/20161201_150512_US5834_createcrdsdepositexportpreviewreportinmp.sql]: turn Insert_Identity Off

File: CI/SQL/20161201_150512_US5834_createcrdsdepositexportpreviewreportinmp.sql [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+GO
+
+DECLARE @report_id int;
+DECLARE @page_id int;
+
+SELECT @page_id = p.Page_ID FROM
+	[dbo].[dp_Pages] p WHERE p.Display_Name = N'Deposits'
+
+
+IF NOT EXISTS(SELECT * FROM [dbo].[dp_reports] WHERE Report_Path = '/MPReports/Crossroads/CRDS Selected Deposit Export Preview')
```
Comment (silbermm) [inline CI/SQL/20161201_150512_US5834_createcrdsdepositexportpreviewreportinmp.sql]: check on ID

File: CI/SQL/20161201_150512_US5834_createcrdsdepositexportpreviewreportinmp.sql [hunk 3]
```diff
@@ -0,0 +1,48 @@
...
+			   ,N'Selected Deposit - Export Preview'
+			   ,N'Selected Deposit - Export Preview (Donations and Payments)'
+			   ,N'/MPReports/Crossroads/CRDS Selected Deposit Export Preview'
+			   ,1
+			   ,1          
+			   ,0)
+END
+
+SELECT @report_id = r.Report_ID FROM
+	[dp_reports] r WHERE r.Report_Path = N'/MPReports/Crossroads/CRDS Selected Deposit Export Preview'
```
Comment (silbermm) [inline CI/SQL/20161201_150512_US5834_createcrdsdepositexportpreviewreportinmp.sql]: use the id that you used above...

Comment (lknair2015) [inline CI/SQL/20161201_150512_US5834_createcrdsdepositexportpreviewreportinmp.sql]: done

File: CI/SQL/20161201_150512_US5834_createcrdsdepositexportpreviewreportinmp.sql (hunk 2 above)
Comment (lknair2015) [inline CI/SQL/20161201_150512_US5834_createcrdsdepositexportpreviewreportinmp.sql]: done

**Pull request merged**